extensions/gadgets/AdviceBar/AdviceBar.html

```diff
@@ -1,3 +1,5 @@
+<script src="{{cache_slug}}/extensions/gadgets/AdviceBar/AdviceBar.js"></script>
```

Comment:
It's not based on the directory. Both will return `/build/cache_slug` when in prod. The difference is that `ASSET_DIR_PREFIX` is a jinja variable and can be interpolated in files that are directly rendered by jinja. On the other hand, `AdviceBar.html` is itself inlined using a jinja variable, so any jinja variables in it need to be interpolated using `interpolate_cache_slug()` in jina_utils.py